src/h3lib/lib/h3Index.c

```diff
@@ -515,30 +518,29 @@ int H3_EXPORT(maxUncompactSize)(const H3Index* compactedSet, const int numHexes,
         } else {
             // Bigger hexagon to reduce in size
             int numHexesToGen =
-                H3_EXPORT(maxH3ToChildrenSize)(compactedSet[i], res);
+                H3_EXPORT(maxCellToChildrenSize)(compactedSet[i], res);
             maxNumHexagons += numHexesToGen;
         }
     }
     return maxNumHexagons;
 }
 
 /**
- * h3IsResClassIII takes a hexagon ID and determines if it is in a
+ * isResClassIII takes a hexagon ID and determines if it is in a
  * Class III resolution (rotated versus the icosahedron and subject
  * to shape distortion adding extra points on icosahedron edges, making
  * them not true hexagons).
  * @param h The H3Index to check.
  * @return Returns 1 if the hexagon is class III, otherwise 0.
  */
-int H3_EXPORT(h3IsResClassIII)(H3Index h) { return H3_GET_RESOLUTION(h) % 2; }
+int H3_EXPORT(isResClassIII)(H3Index h) { return H3_GET_RESOLUTION(h) % 2; }
```

Comment:
Seems like this should accept a resolution rather than an index.

Reply:
So this is letting people know if the hexagon is a class III hexagon. I would prefer it to just be `isClassIII` and return `isResDigitClassIII` to this name, but that isn't what we agreed upon in the RFC and I think the relitigation of these decisions all the time is why 4.0 is taking so long to get out the door. :/

Reply:
This function name is probably fine to keep. I don't think it blocks this PR

Reply:
I'm happy to revisit in a follow-up. I could debate names all day 😛 .

I think these changes have taken so long just due to the scale and the coupling between name changes. After this PR lands, I think @dfellis has done most of the hard work. I'd expect follow-up adjustments like this to be smaller-scale and less-coupled, so we should be able to resolve them fairly quickly. 🙏